src/utils.js

```diff
@@ -575,3 +575,16 @@ export const formatCode = (code, errorLines) => {
   })
   return '    ' + lines.join('\n    ')
 }
+
+/**
+ * Pipe subprocess output to stdio
+ *
+ * @param {import('execa').ExecaChildProcess} subprocess
+ * @param {string} prefix
+ * @param {boolean} [noPrefix]
+ */
+export function pipeOutput (subprocess, prefix, noPrefix) {
+  prefix = noPrefix ? '' : kleur.gray(prefix + ': ')
+  subprocess.stdout?.on('data', (data) => process.stdout.write(`${prefix}${data}`))
```

Comment:
If `data` contains newlines I think this needs to add the prefix to each line:

```suggestion
  subprocess.stdout?.on('data', (data) => {
    process.stdout.write(
      data.toString('utf8')
        .split('\n')
        .map((/** @type {string} */ line) => `${prefix}${line}`)
        .join('\n')
    )
  })
```

..same for `stderr`